clients/src/main/java/org/apache/kafka/common/ConsumerGroupState.java

```diff
@@ -31,7 +31,15 @@ public enum ConsumerGroupState {
     COMPLETING_REBALANCE("CompletingRebalance"),
     STABLE("Stable"),
     DEAD("Dead"),
-    EMPTY("Empty");
+    EMPTY("Empty"),
+
+    // New consumer group states are temporary here. It will be removed when
+    // the KIP-848 is completed.
+    NEW_CONSUMER_GROUP_EMPTY("empty"),
```

Comment:
So it was decided that we would make the state filters case insensitive, which means that for states that are common in both the old protocol and the new protocol, we return all groups in that state. Ex: "Stable" returns are groups that are stable. The enum will therefore contain:
`public enum ConsumerGroupState {
    UNKNOWN("Unknown"),
    PREPARING_REBALANCE("PreparingRebalance"),
    COMPLETING_REBALANCE("CompletingRebalance"),
    STABLE("Stable"),
    DEAD("Dead"),
    EMPTY("Empty"),
    ASSIGNING("Assigning"),
    RECONCILING("Reconciling");
}`

Reply:
We would also have to add support to make the filter case insensitive both in the command line code and in both the group coordinators list groups methods

Reply:
done